v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -74,12 +85,12 @@ const responseBuilder = async (metadata, message, { Config }, payload) => {
       const filteredLoginCustomerId = removeHyphens(Config.loginCustomerId);
       response.headers["login-customer-id"] = filteredLoginCustomerId;
     } else
-      throw new ErrorBuilder()
-        .setMessage(
-          `[Google_adwords_enhanced_conversions]:: loginCustomerId is required as subAccount is true.`
-        )
-        .setStatus(400)
-        .build();
+      throw new TransformationError(
+        `[Google_adwords_enhanced_conversions]:: loginCustomerId is required as subAccount is true.`,
```

Comment:
[Google_adwords_enhanced_conversions]:: requires since we have dest_type?

Reply:
We don't need it, just made changes to classes. Will do these changes

Reply:
Done